flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/resource/ResourceManager.java

```diff
@@ -65,6 +69,9 @@ public class ResourceManager implements Closeable {
     private static final String FILE_SCHEME = "file";
 
     private final Path localResourceDir;
+    /** Resource infos for functions. */
+    private final FunctionResourceManager functionResources;
```

Comment:
Do we need to introduce a FunctionResourceManager. Actually, the ResourceManager doesn't distinguish the jar is from Funciton or Connector. I think we can just move the unregister method from the `FunctionResourceManager` to here.